lib/Jit/LLILCJit.cpp

```diff
@@ -641,3 +641,12 @@ void ObjectLoadListener::getRelocationTypeAndAddend(uint8_t *FixupAddress,
     llvm_unreachable("Unknown reloc type.");
   }
 }
+
+unsigned LLILCJit::getMaxIntrinsicSIMDVectorLength(DWORD CpuCompileFlags) {
+  return GenIR::getPreferIntrinsicSIMDVectorLength();
+}
+
+bool LLILCJitContext::isAltJit() {
+  JitOptions JitOptions(*this);
+  return JitOptions.IsAltJit;
```

Comment:
When the EE loads the System.Numerics.Vectors assembly, it asks the JIT what length (in bytes) of SIMD vector it supports as an intrinsic type.
LLILC has to know who determines Generic Vector Size and return RyuJit value or own.